app.js

```diff
@@ -1,8 +1,104 @@
 // import functions and grab DOM elements
+import { renderPoll } from './render-poll.js';
 
+const optionAYesEl = document.getElementById('option-a-yes');
+const optionBYesEl = document.getElementById('option-b-yes');
+
+const optionANoEl = document.getElementById('option-a-no');
+const optionBNoEl = document.getElementById('option-b-no');
+
+const archivePollEl = document.getElementById('archive-poll');
+const currentPollEl = document.getElementById('newest-poll');
+const pastPollEl = document.getElementById('past-polls');
+
+const pollForm = document.getElementById('poll-form');
 // let state
+let pastPolls = [];
+
+let question = '';
+let optionA = '';
+let optionB = '';
+let votesA = 0;
+let votesB = 0;
+
 
 // set event listeners 
   // get user input
   // use user input to update state 
   // update DOM to reflect the new state
+pollForm.addEventListener('submit', (event) => {
+    currentPollEl.textContent = '';
+    event.preventDefault();
+    votesA = 0;
+    votesB = 0;
+    const data = new FormData(pollForm);
+
+    const userQuestion = data.get('question');
+    const userOptionA = data.get('option-a');
+    const userOptionB = data.get('option-b');
+
+    question = userQuestion;
+    optionA = userOptionA;
+    optionB = userOptionB;
+
+  
+    displayCurrentPoll(); 
+});
+
+optionAYesEl.addEventListener('click', () => {
+    votesA++;
+    renderPoll();
+    displayCurrentPoll();
+});
+optionBYesEl.addEventListener('click', () => {
+    votesB++;
+    renderPoll();
+    displayCurrentPoll();
+});
+optionANoEl.addEventListener('click', () => {
+    votesA--;
+    renderPoll();
+    displayCurrentPoll();
+});
+optionBNoEl.addEventListener('click', () => {
+    votesB--;
+    renderPoll();
+    displayCurrentPoll();
+});
+
+archivePollEl.addEventListener('click', () => {
+
```

Comment:
You'll want to empty out the contents of your current poll div somewhere in here so that when someone archives the current poll it resets the DOM elements that display the current poll